python/triton/language/core.py

```diff
@@ -871,7 +871,7 @@ def __not__(self, _builder=None):
 
     @builtin
     def __getitem__(self, slices, _builder=None):
-        if isinstance(slices, (slice, constexpr)):
+        if isinstance(slices, (slice, constexpr)) or slices is None:
```

Comment:
I don't understand why this is correct -- can you add a comment to the code?

Reply:
In the interpreter mode, constant values and immediate numbers are not wrapped in `constexpr`. Therefore `None` is just `None`

Reply:
Maybe I should add a comment in the commit log? Otherwise, there are a lot of `_constexpr_to_value` conversions

Reply:
I feel like a comment on just one of them that says "here and below" would be fine, it's easy to see.

Reply:
Got it. No problem